src/components/NotificationsDrawer/DrawerPanelContent.tsx

```diff
@@ -1,16 +1,34 @@
-import React from 'react';
-import { Button } from '@patternfly/react-core/dist/dynamic/components/Button';
-import { EmptyState, EmptyStateBody, EmptyStateIcon } from '@patternfly/react-core/dist/dynamic/components/EmptyState';
-import { NotificationDrawer, NotificationDrawerHeader } from '@patternfly/react-core/dist/dynamic/components/NotificationDrawer';
-import { Text } from '@patternfly/react-core/dist/dynamic/components/Text';
-import { Title } from '@patternfly/react-core/dist/dynamic/components/Title';
+import React, { useEffect, useState } from 'react';
+import {
+  Badge,
+  Button,
+  Checkbox,
+  Divider,
+  Dropdown,
+  DropdownGroup,
+  DropdownItem,
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  Icon,
+  MenuToggle,
+  MenuToggleElement,
+  NotificationDrawer,
+  NotificationDrawerBody,
+  NotificationDrawerHeader,
+  NotificationDrawerList,
+  Text,
+  Title,
+} from '@patternfly/react-core';
```

Comment:
Do not override ethe dynamic imports. They are used to fix tree-shaking issues for module federation environments.